apps/jenkins/jenkins/ptl-intsvc/jenkins-azure-vm-agent.yaml

```diff
@@ -113,4 +113,18 @@ spec:
                           azureVMCloudRetentionStrategy:
                             idleTerminationMinutes: 5
                         usageMode: "EXCLUSIVE"
+                        virtualMachineSize: "Standard_D4ds_v5"
+                        <<: *vm_template_values_anchor
+                      - templateName: "cnp-jenkins-builders-arm"
+                        templateDesc: "Jenkins build agents running on ARM64 CPUs"
+                        labels: "daily arm"
```

Comment:
the main pool shouldn't have daily label on it, CFT have this split out to a different one so daily builds can't block the queue